easybuild/easyconfigs/p/pigz/pigz-2.4-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,51 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+#
+# 2.4.0:
+# Jonas Demeulemeester
+# The Francis Crick Institute
+#
+# This one is modified from the pigz-2.4-GCCcore-7.3.0.eb. The download link in 
+# pigz-2.3.4-GCCcore-6.4.0.eb does not exist anymore.
```

Comment:
@murfreesboro The 2nd part of this comment shouldn't be mentioned here.

Instead, please open an issue to report this problem (via https://github.com/easybuilders/easybuild-easyconfigs/issues/new), or open a pull request to fix the source URL in the existing `pigz` easyconfigs

Reply:
Okay. I opened an issue to report this problem. Also deleting the corresponding comment.